config/src/config/state_sync_config.rs

```diff
@@ -332,8 +339,10 @@ impl Default for AptosDataClientConfig {
             max_subscription_lag_secs: 30, // 30 seconds
             max_transaction_chunk_size: MAX_TRANSACTION_CHUNK_SIZE,
             max_transaction_output_chunk_size: MAX_TRANSACTION_OUTPUT_CHUNK_SIZE,
-            optimistic_fetch_timeout_ms: 5000,        // 5 seconds
-            response_timeout_ms: 10_000,              // 10 seconds
+            min_peer_ratio_for_latency_filtering: 5, // Only filter if we have at least 5 potential peers per request
+            min_peers_for_latency_filtering: 10, // Only filter if we have at least 10 total peers
```

Comment:
How many peers do our mainnet PFNs have?

Reply:
They are currently configured to 4 😄 (That's been the default for a while). But, we should probably revisit that now that things are changing.

Reply:
Somewhat orthogonal to this PR, but I'm concerned that with our 7 peer forge networks, we don't really exercise the numbers that mainnet would see.